app/packages/core/src/components/Actions/ActionsRow.tsx

```diff
@@ -431,13 +431,13 @@ export const GridActionsRow = () => {
   return (
     <ActionsRowDiv>
       <ToggleSidebar modal={false} />
-      <Options modal={false} />
       <Colors />
       {hideTagging ? null : <Tag modal={false} />}
       <Patches />
       {!isVideo && <Similarity modal={false} />}
       <SaveFilters />
       <Selected modal={false} />
+      <Options modal={false} />
```

Comment:
what is this for? 🤔

Reply:
moving the options all the way over so its on the far right, from the document you tagged me in : 

_Move the settings gear in the grid’s action menu to the far right, to avoid confusion with the new settings gear in the sidebar_

Reply:
oh right! cool cool